apps/crl.c

```diff
@@ -286,7 +286,7 @@ int crl_main(int argc, char **argv)
             if (crlnumber == i) {
                 ASN1_INTEGER *crlnum;
                 crlnum = X509_CRL_get_ext_d2i(x, NID_crl_number, NULL, NULL);
-                BIO_printf(bio_out, "crlNumber=");
+                BIO_printf(bio_out, "crlNumber=0x");
```

Comment:
You would need to do a seperate BIO_printf() inside the if otherwise we will end up with..
crlNumber=0x<None>